CHANGELOG.md

```diff
@@ -29,6 +29,10 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 - **CUMULUS-1682**
   - Moved all `@cumulus/ingest/parse-pdr` code into the `parse-pdr` task as it had become tightly coupled with that task's handler and was not used anywhere else. Unit tests also restored.
+- **CUMULUS-1777**
```

Comment:
Add breaking changes or migration steps section for recommended deletion of table.